README.md

```diff
@@ -24,5 +24,13 @@ extensions:
 tasks:
   drupalcheck:
     drupal_root: ~
+    deprecations: true
+    analysis: true
+    php8: true
 ```
-Optionally, configure the path to the Drupal root. This fallback option can be used if drupal-check could not identify Drupal root from the provided path(s). This is useful when testing a module as opposed to a Drupal installation.
+Optionally, you can define multiple DrupalCheck arguments:
+
+- **drupal_root** (string): Configure the path to the Drupal root. This fallback option can be used if drupal-check could not identify Drupal root from the provided path(s). This is useful when testing a module as opposed to a Drupal installation.
+- **deprecations** (boolean): Check code for deprecations.
```

Comment:
If we apply the suggestion from here: https://github.com/Metadrop/grumphp-drupal-check/pull/9#discussion_r1085588523; then we can specify here that it will be true by default.

```suggestion
```

```suggestion
- **deprecations** (boolean): Check code for deprecations. By default it is true.
```